docs/help/Binche.md

```diff
@@ -67,4 +67,10 @@ Newbies may skip the bold steps when first trying to understand the scenario.
 * 1:33 "_animate_" pair diagram
 * 1:36 increase thread diagram container (alternative: ![](../images/size-inc.jpg) ![](../images/size-dec.jpg))
 * 1:38 scroll down to get thread container back into view
-* 1:42 toggle thread color when tooltip displays thread number
\ No newline at end of file
+* 1:42 toggle thread color when tooltip displays thread number
+
+### History of Ground 31
+
+In issue 2000/2 of the Belgian lace magazine [_Kant_](https://www.kantcentrum.eu/en/magazine), Anne-Marie Verbeke-Billiet published a handkerchief pattern called _Anneke_ based on an old Valenciennes example.  She included four variations of stitches traditionally used in Paris ground and in the ring pairs around snowflakes.  The variation that is most unusual uses a combination of whole (CTCT) and half (CT) stitches.  This stitch combination, which she labels TT4 in the article (see also B4.2 in _Viele gute Gründe_ by Ulricke Voelcker-Löhr), has the effect of blurring the ring pairs around the snowflakes until they almost disappear.  According to Verbeke-Billiet, this stitch combination was once as popular as the whole stitch variation (TT1 in Kant and B4.1 in VGG) that is so familiar today. TT4 fell out of the lacemaker's repertoire about one hundred years ago perhaps because, like many half-stitch grounds, it is very difficult to work. The threads don't travel in tidy vertical and diagonal lines making it hard, especially when worked without pins, to keep track of where you are in the pattern.
+ 
+In 2004, Hildegund Jene published a sampler called [_Binche Kompakt_](https://www.deutscher-kloeppelverband.de/katalog/) which featured 32 different grounds drawn from Flemish, Parisian,and Valenciennes laces.  Ground number 31 is the challenging TT4 ground described by Verbeke-Billiet.  When Jo Pol started to work on Binche Kompakt, she could find neither a thread diagram, nor a high resolution photo of a sample of TT4.  She decided to create a thread diagram using GroundForge.  This exercise motivated Pol to include several of the features you find in GroundForge today.  
```

Comment:
You are repeating line 10.

Reply:
Line 10 is a much shorter version of line 76.  Since this history comes at the bottom of the web page and explains the background, I think it is still of interest and worth keeping.